stdlib/tkinter/__init__.pyi

```diff
@@ -452,14 +481,14 @@ class CallWrapper:
     def __call__(self, *args): ...
 
 class XView:
-    def xview(self, *args): ...
-    def xview_moveto(self, fraction): ...
-    def xview_scroll(self, number, what): ...
+    def xview(self) -> Tuple[float, float]: ...
```

Comment:
It is possible to run `foo.xview('moveto', 0.5)`, and scrollbars usually call `xview()` or `yview()` that way. We can add support for that with overloads. Or if you don't want to be repetitive, we can just add a loosely typed overload that gets used when you provide 1 more argument:

```suggestion
    @overload
    def xview(self) -> Tuple[float, float]: ...
    @overload
    def xview(self, *args: Any) -> Any: ...
```

Reply:
Thanks, it hadn't occurred to me that the scrollbar-callback convention would use `xview()` or `yview()` in this way. I don't know if MyPy is smart enough to type-infer through layers of callbacks and potentially into the Tcl runtime, but I guess explicit is better than implicit.

Reply:
Currently it accepts any function that accepts two floats as arguments: https://github.com/python/typeshed/blob/b9e1d7d522fe90b98e07d43a764bbe60216bc2c4/stdlib/tkinter/__init__.pyi#L110